AzureADSample/README.md

```diff
@@ -17,4 +17,10 @@ Built in .NET MAUI and used Azure AD for Auth Service. This sample only demonstr
 
 ## Nugget Packages
 - `Microsoft.Identity.Client`
-- `System.IdentityModel.Tokens.Jwt`
\ No newline at end of file
+- `System.IdentityModel.Tokens.Jwt`
+
+For more information about the sample see:
+- [Authentication with Azure AD B2C] (https://www.youtube.com/watch?v=sTPWF2O456U)
+- [Set up Google - Azure AD B2C] (https://docs.microsoft.com/en-us/azure/active-directory-b2c/identity-provider-github?WT.mc_id=Portal-Microsoft_AAD_B2CAdmin&pivots=b2c-user-flow)
+- [Set up Facebook - Azure AD B2C] (https://docs.microsoft.com/en-us/azure/active-directory-b2c/identity-provider-facebook?WT.mc_id=Portal-Microsoft_AAD_B2CAdmin&pivots=b2c-user-flow)
+- [Set up Twitter - Azure AD B2C] (https://docs.microsoft.com/en-us/azure/app-service/configure-authentication-provider-twitter)
```

Comment:
Did you intend to add this to the README for the Azure AD B2C sample instead of this one (the AzureADSample)?

Reply:
@gal42 bumping this question!

Reply:
Yes! I moved it yesterday but it seems that I did not delete it from the Azure AD Docs, ups! (It's fixed now)